testsuite/forge/src/success_criteria.rs

```diff
@@ -616,25 +686,37 @@ impl SuccessCriteriaChecker {
         max_failed_submission_config: Option<f64>,
         stats_rate: &TxnStatsRate,
         traffic_name_addition: &String,
-    ) -> anyhow::Result<()> {
-        collect_errors!(
+    ) -> anyhow::Result<(), SuccessCriteriaErrors> {
+        let mut runner = SuccessCriteriaResults::new();
+
+        runner.add_result(
+            "Min avg tps",
+            CheckType::Soft,
```

Comment:
Should this be hard? We need to ensure that the test runner itself also doesn't regress/we are sending enough traffic, though idk if this is the best place to do that.

Reply:
In the spirit of soft failing on metric violations, I want to leave this Soft. We will think about how we address that we are indeed making sure we action on regressions.